projects/fourier/lib/fourier/utilities/xcode.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "json"
+
+module Fourier
+  module Utilities
+    module Xcode
+      def self.current_xcode_version
+        %x{ xcode-select -p }.split(/(?<=app)/).first
+      end
+
+      def self.switch_xcode_version(xcode_path)
+        Utilities::System.system("sudo xcode-select -switch #{xcode_path}")
+      end
+
+      def self.path_to_xcode(version)
```

Comment:
I'd like to document this method because it's complex, but I'm not sure what documentation looks like in Ruby. Are there any good examples in the app I could reference?